web-ui/src/components/admin/roles/Roles.jsx

```diff
@@ -62,10 +62,30 @@ const Roles = () => {
   memberProfiles?.sort((a, b) => a.name.localeCompare(b.name));
 
   useEffect(() => {
-    if (roles) {
-      setSelectedRoles(roles.map(roleObj => roleObj.role));
-    }
-  }, [roles]);
+    const url = new URL(location.href);
+    setSearchText(url.searchParams.get('searchText') ?? '');
+    const selectedRoles = url.searchParams.get('selectedRoles');
+    const roles = selectedRoles?.split(',') ?? [];
+    if (roles.length === 0) roles.push('ADMIN', 'MEMBER', 'PDL');
```

Comment:
I don't think we ever want to start with no roles selected. So if none are present in the query parameter then we make them all selected.